includes/class-learndasheddgift.php

```diff
@@ -238,11 +238,7 @@ public function plugin_activation_handler() {
 		if ( ! wp_next_scheduled( 'emails_gift_reminder' ) ) {
 			$date         = gmdate( 'Y-m-d' );
 			$current_time = strtotime( $date . ' 07:00:00' );
-			if ( $current_time > time() ) {
-				wp_schedule_event( $current_time, 'wdm_daily_emails_gift_reminder', 'emails_gift_reminder' );
-			} else {
-				wp_schedule_event( strtotime( '+1 day', $current_time ), 'wdm_daily_emails_gift_reminder', 'emails_gift_reminder' );
-			}
+			wp_schedule_event( $current_time, 'wdm_emails_gift_reminder', 'emails_gift_reminder' );
```

Comment:
I know this is how it was before, but any reason it is getting the current time like this? Seems like getting the actual current time would be better.

Reply:
None that I can see. time() is used in the docs and that's good enough for me.